<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>574</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>444</width>
    <height>639</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/mail.png</normaloff>../resources/mail.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_principal">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(233, 242, 249);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_esquerda">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>444</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>24</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(27, 50, 95);</string>
           </property>
           <property name="text">
            <string>Bem vindo ao Appostman</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>24</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>Fazer login</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_login_username">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Palatino Linotype</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
background-color: rgb(212, 220, 226);
border: 1px solid;
border-radius:5px
}


QLineEdit:hover
{
background-color: rgb(199, 206, 211);
}</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Usu√°rio</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_login_password">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Palatino Linotype</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
background-color: rgb(212, 220, 226);
border: 1px solid;
border-radius:5px
}


QLineEdit:hover
{
background-color: rgb(199, 206, 211);
}</string>
           </property>
           <property name="maxLength">
            <number>30</number>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Senha</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="pushButton_login">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Palatino Linotype</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: rgb(158, 195, 255);
border: 1px solid;
border-radius:5px
}

QPushButton:hover
{
background-color: rgb(212, 220, 226);
color: rgb(0, 0, 0);
}</string>
              </property>
              <property name="text">
               <string>Entrar</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>resources/circle-right-arrow.png</normaloff>resources/circle-right-arrow.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>180</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
             <widget class="QPushButton" name="pushButton_redefine_password">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>170</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Rockwell</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: rgba(0, 0, 0, 0);
	color: rgb(58, 137, 201);

}

QPushButton:hover
{
	color: rgb(156, 196, 228);
}</string>
              </property>
              <property name="text">
               <string>Esqueceu a senha?</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="pushButton_newuser">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Rockwell</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: rgba(0, 0, 0, 0);
	color: rgb(58, 137, 201);

}

QPushButton:hover
{
	color: rgb(156, 196, 228);
}</string>
              </property>
              <property name="text">
               <string>Ainda n√£o possui uma conta?</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Rockwell</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Vers√£o 1.0/2023</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_direita">
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>20</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(27, 50, 95);</string>
           </property>
           <property name="text">
            <string>Seu software de e-mail marketing!</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>20</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>Criar conta</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_signup_username">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Palatino Linotype</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
background-color: rgb(212, 220, 226);
border: 1px solid;
border-radius:5px
}


QLineEdit:hover
{
background-color: rgb(199, 206, 211);
}</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Usu√°rio</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_signup_email">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Palatino Linotype</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
background-color: rgb(212, 220, 226);
border: 1px solid;
border-radius:5px
}


QLineEdit:hover
{
background-color: rgb(199, 206, 211);
}</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_signup_password">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Palatino Linotype</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
background-color: rgb(212, 220, 226);
border: 1px solid;
border-radius:5px
}


QLineEdit:hover
{
background-color: rgb(199, 206, 211);
}</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Senha</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_signup_repeat_password">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Palatino Linotype</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
background-color: rgb(212, 220, 226);
border: 1px solid;
border-radius:5px
}


QLineEdit:hover
{
background-color: rgb(199, 206, 211);
}</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Digite a senha novamente</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>61</number>
            </property>
            <item>
             <widget class="QCheckBox" name="checkBox_receive_emails">
              <property name="font">
               <font>
                <family>Palatino Linotype</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Aceito receber e-mails informativos</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButton_signup">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Palatino Linotype</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: rgb(158, 195, 255);
border: 1px solid;
border-radius:5px
}

QPushButton:hover
{
background-color: rgb(212, 220, 226);
color: rgb(0, 0, 0);
}</string>
              </property>
              <property name="text">
               <string>Cadastrar-se</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>resources/add-contact.png</normaloff>resources/add-contact.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="pushButton_goback">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Rockwell</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: rgba(0, 0, 0, 0);
	color: rgb(58, 137, 201);

}

QPushButton:hover
{
	color: rgb(156, 196, 228);
}</string>
              </property>
              <property name="text">
               <string>Voltar ao Login</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>label_2</tabstop>
  <tabstop>lineEdit_login_username</tabstop>
  <tabstop>lineEdit_login_password</tabstop>
  <tabstop>lineEdit_signup_username</tabstop>
  <tabstop>lineEdit_signup_email</tabstop>
  <tabstop>lineEdit_signup_password</tabstop>
  <tabstop>lineEdit_signup_repeat_password</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
